sites/postcss-preset-env/blog/can-use-boolean-custom-media.md

```diff
@@ -0,0 +1,35 @@
+---
+title: You can now use true and false in custom media queries!
+description: This allows you to easily debug a condition without a major refactor.
+date: 2023-01-25
+---
+
+Since the release of PostCSS Custom Media version 9, you can now use `true` and `false` in your custom media queries. This allows you to easily debug a condition without a major refactor.
+
+```css
+@custom-media --tablet (width >= 48rem);
+@custom-media --desktop (width >= 90rem);
+@custom-media --dark true;
+
+@media (--tablet) and (--dark) {
+  body {
+    color: cyan;
+  }
+}
+```
+
+Now you can easily test what the media query would look like if you were actually in Dark Mode. While there are developer tools that can help you with this, they lose the state if you close the for example or you might want to test another combination. 
+
+Just remember to remove the `true` and `false` values before you ship your code!
```

Comment:
Should this be more visually clear that it is source code?

<img width="259" alt="Screenshot 2023-02-05 at 23 16 45" src="https://user-images.githubusercontent.com/11521496/216849100-40ef6f26-99ab-4454-8a8c-3d0baee4add7.png">

Reply:
Completely agree!

Now looks like this:

![Screenshot 2023-02-06 at 07 45 53](https://user-images.githubusercontent.com/946645/216901788-aead9a82-485d-4ca6-8f29-d9eb4919b7ea.png)

Reply:
nice!